test/parallel/test-worker-cli-options.js

```diff
@@ -1,16 +1,30 @@
-// Flags: --expose-internals
+// Flags: --expose-internals --expose-gc
```

Comment:
Shouldn't we validate whether `gc()` is available in the Worker global scope?

Reply:
I have added the `global.gc()` to worker's code, please helo review again.